### PR TITLE
feat : Panel container get rest props

### PR DESCRIPTION
**Pull request by xyuanbuilds**
Panel container can get event handler like mouseover, mouseleave

### CODE REVIEW

**Review by yoyo837 — APPROVED** [review]

**Comment by yoyo837** [comment]
CI failed.

**Comment by xyuanbuilds** [comment]
> CI failed.

我去增加个测试吧

**Comment by xyuanbuilds** [comment]
> CI failed.

O 了

File: src/Panel.tsx [hunk 1]
```diff
@@ -118,7 +119,7 @@ class CollapsePanel extends React.Component<CollapsePanelProps, any> {
     const ifExtraExist = extra !== null && extra !== undefined && typeof extra !== 'boolean';
 
     return (
-      <div className={itemCls} style={style} id={id}>
+      <div className={itemCls} style={style} id={id} {...rest}>
```
Comment (yoyo837) [inline src/Panel.tsx:122]: ...rest放前面吧？保险一点。哪天动props的时候不至于className等失效

File: tests/index.spec.tsx [hunk 2]
```diff
@@ -598,4 +598,17 @@ describe('collapse', () => {
...
+    const wrapper = mount(
+      <Collapse defaultActiveKey="1">
+        <Panel header="collapse 1" key="1" onClick={() => (window.numTest = 100)}>
+          <div className="target">Click this</div>
+        </Panel>
+      </Collapse>,
+    );
+    expect(window.numTest).toBe(undefined);
+    wrapper.find('.target').simulate('click');
+    expect(window.numTest).toBe(100);
```
Comment (yoyo837) [inline tests/index.spec.tsx:612]: 用`expect(func).toHaveBeenCalled();`这种形式

**Pull request merged**